### ISSUE TITLE
Exams: update exam card for edx.org

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#4636]
#### Current Text:

Final Proctored Exam

You must take a proctored exam for each course. Exams may be taken at any authorized Pearson test center. Before you can take an exam, you have to pay for the course and pass the online work.

[gray block with profile information]

#### New Text:

Final Proctored Exam

You must take an online proctored exam for each course. Before you can take an exam, you have to pay for the course and pass the online work.

Exams will be available online on edX.org. You may take the exam at any time during the exam period. No advance scheduling is required, but you should verify your account and complete the exam onboarding as soon as possible. 

You are authorized to take the following exams, before [date] [time] [time zone]"

•  list of links to each approved exam 

## Acceptance Criteria:

- [ ] Update the text
- [ ] Remove the block about profiles and addresses. These are no longer necessary
- [ ] provide a link for each course that links to the appropriate course on edX and includes an embedded enrollment code

#### Out of Scope

- update course cards

**Comment by annagav** [comment]
@pdpinch [the third checkbox] I think the link to each exam should probably appear in the course row, with some text navigating the user to take the exam. As I understand the link to the exam would appear only after the user got a final grade for the course and has paid for it. Are enrollment codes generated by the course team?

UI option: the link to the exam could appear in a form of a link or button(similar to PAY button).

**Comment by pdpinch** [comment]
That’s a really good idea. If I recall correctly, the “exam card” was designed originally because Pearson would let us direct-link into each exam. We can do that now. 

We know from customer support tickets that many users are confused, looking for an exam link on the course card and not noticing the link in the exam card.

**Comment by annagav** [comment]
The proposed steps for setting up exams:
- create ExamRuns (manually),
- authorize exams runs (scheduled task)
- populate the ExamAuthorizations with coupon codes (a management command?)
Assumptions: ExamProfile `status` doesn't seem to change any behavior, so no modification here.


I am thinking we might need the messages like:
"You did not pass the exam. Click here to take an exam again."
Or a message for when we don't have the coupon codes ready yet:
"There are currently no exams available. Please check back later."
